tests/queries/0_stateless/02495_parser_string_binary_literal.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+$CLICKHOUSE_CLIENT --query="SELECT b '0';" 2>&1 | grep -o 'Syntax error'
```

Comment:
I realized that in some cases syntax errors in `sql` tests not properly handled, so I moved this cases into `sh` test

Reply:
@azat 
Yes, exactly

https://s3.amazonaws.com/clickhouse-test-reports/43785/e88b85178fd33446ab76ee7477dd2b69a77fea7c/fast_test.html

```
2022-12-11 08:55:21 Code: 62. DB::Exception: Syntax error: failed at position 8 ('b'3'; -- { clientError SYNTAX_ERROR }
2022-12-11 08:55:21 SELECT x'k'; -- { clientError SYNTAX_ERROR }
2022-12-11 08:55:21 
2022-12-11 08:55:21 SELECT b'1; -- { clientError SYNTAX_ERROR }
2022-12-11 08:55:21 SELECT x'a; -- { clientError SYNTAX_ERROR }
2022-12-11 08:55:21 
2022-12-11 08:55:21 SELECT b"0"; -- { clientError SYNTAX_ERROR }
2022-12-11 08:55:21 SELECT x"0"; -- { clientError SYNTAX_ERROR }
2022-12-11 08:55:21 ') (line 1, col 8): b'3'; -- { clientError SYNTAX_ERROR }
...
```

Reply:
So, does it mean that I should  just remove the existing sql tests and only leaves the sh one?